pkg/ivd/ivd_protected_entity.go

```diff
@@ -291,7 +291,7 @@ func (this IVDProtectedEntity) Snapshot(ctx context.Context, params map[string]m
 	this.logger.Infof("CreateSnapshot called on IVD Protected Entity, %v", this.id.String())
 	var retVal astrolabe.ProtectedEntitySnapshotID
 	retryInterval := time.Second
-	retryCount := 0
+	retryCount := -1
```

Comment:
What does this mean?  retry indefinitely?

Reply:
I want to track only the retries, not the first invocation, hence the -1. If set to 0, in the preceding log msg about retry makes it seem like it was retried once.